articles/api/authentication/api-authz/_highly-regulated.md

```diff
@@ -0,0 +1,213 @@
+# Highly Regulated Identity
+## Push Authorization Requests (PAR)
+
+<%= include('../../../_includes/_http-method', {
+  "http_badge": "badge-primary",
+  "http_method": "POST",
+  "path": "/oauth/par",
+  "link": "##push-authorization-requests-par-"
+}) %>
+
+```http
+POST ${account.namespace}/oauth/par
+Content-Type: 'application/x-www-form-urlencoded'
+  response_type=code|code id_token&
```

Comment:
It is important that all example include
* an audience param (optional)
* an authorization_details param (optional)

Reply:
We only have Required and Recommended tags for the Explorer, but I can add these params.

Reply:
Pending updates on Google doc